docs/user/03-60-formatting-service-binding-secret.md

```diff
@@ -0,0 +1,232 @@
+# Formats of Service Binding Secrets
+
+Use different attributes in your ServiceBinding resource to generate different formats of your Secret resources.
+
+## Overview
+
+Secret resources share a common set of basic parameters that can be divided into two categories:
+* Credentials returned from the service broker that allow your application to access and consume an SAP BTP service.
+* Attributes of the associated service instance: The details of the service instance itself.
+
+However, the Secret resources can come in various formats:
+* Default key-value pairs
+* A JSON object
+* One JSON object with credentials and service information
+* Custom formats
+
+
+## Key-Value Pairs
+
+If you do not use any of the attributes, the generated Secret is by default in the key-value pair format, as in the following examples:
+
+* Service binding
+
+  ```yaml
+  apiVersion: services.cloud.sap.com/v1
+  kind: ServiceBinding
+  metadata:
+    name: my-binding
+  spec:
+    serviceInstanceName: my-service-instance
+  ```
+* Secret
+
+  ```yaml
+  apiVersion: v1
+  metadata:
+    name: my-binding
+  kind: Secret
+  stringData:
+    uri: https://my-service.authentication.eu10.hana.ondemand.com
+    client_id: admin
+    client_secret: ********
+    instance_guid: my-sample-instance-guid // The service instance ID
+    instance_name: my-service-instance // Taken from the service instance external_name field if set. Otherwise from metadata.name
+    plan: sample-plan // The service plan name                
+    type: sample-service  // The service offering name
+  ```
+
```

Comment:
If formatted as task topic, everything above this line is "Context", while everything below are instructions - how to switch from the default setup to another format. 

The instructions are probably alternatives to each other, not a sequence of steps, but in DITA, that could be done with the `<steps-unordered>` tag.

You don't have to do this, but it's worth considering.